src/plugins/rename/README.md

```diff
@@ -54,9 +54,28 @@ after the parent key path.
 If an invalid configuration is given or the cut operation would cause a parent key duplicate, the affected keys are simply skipped and not renamed. 
 
 
+## REPLACE ##
+
+Using the `/replacewith` global key or `rename/to`  MetaKey the rename plugin will replace the part removed by `cut` with the supplied String
+
+## TO UPPER / LOWER ##
+
+Using the `/toupper` or `/tolower` global configuration key, or the `rename/toupper` or `rename/tolower` MetaTag the rename plugin will 
+convert the keynames to uppercase or lowercase.
+The supplied values tell the plugin how many levels starting from the right will be converted. `toupper` and `tolower` can be combined.
+When no value or "0" is supplied with the keys the whole name below the parentkey will be converted.
+
+### EXAMPLE ###
+```
+/toupper = 1
+/tolower = 3
```

Comment:
would be nice to get this example, then we can finish this PR

or do you want us to directly test the new INI?